test/stress_tests/rllib_cluster.yaml

```diff
@@ -103,9 +103,9 @@ worker_setup_commands: []
 # Command to start ray on the head node. You don't need to change this.
 head_start_ray_commands:
     - ray stop
-    - ulimit -n 65536; ray start --head --redis-port=6379 --object-manager-port=8076 --autoscaling-config=~/ray_bootstrap_config.yaml
+    - ulimit -n 65536; LD_LIBRARY_PATH=/usr/local/cuda-8.0/lib64 && ray start --head --redis-port=6379 --object-manager-port=8076 --autoscaling-config=~/ray_bootstrap_config.yaml
```

Comment:
1. is there a way to avoid specifying the exact version number? As a convention, there's usually a symlink (e.g., `/usr/local/cuda`) that points to the latest version (e.g., `cuda-8.0`). If we could be version agnostic, that would make the test more future-proof). 
2. You can drop the `&&` after `LD_LIBRARY_PATH`. This sets the env.var and runs `ray start` in the same command.